src/animation_curses.cpp

```diff
@@ -1,4 +1,4 @@
-#if (!defined SDLTILES)
```

Comment:
Fair enough w.r.t noise. It was in the middle of a change that was to affect a lot of files anyway, so at the time, the "why not" made more sense than if I was to only need to touch a small number of files.